Runner/script.sh.template

```diff
@@ -1,11 +1,13 @@
 git clone --no-tags --single-branch --progress https://github.com/dotnet/runtime
 cd runtime
 git log -1
+chmod 777 build.sh
 
 git remote add other https://github.com/{{SOURCE_REPOSITORY}}
 git fetch other {{SOURCE_BRANCH}}
 git log other/{{SOURCE_BRANCH}} -1
-git checkout -b pr
+git switch -c pr
+git reset --hard main
```

Comment:
When would this change anything?

Reply:
The `reset` I mean

Reply:
It's just to be 100% sure the baseline is the same.